dod-advana/gamechanger-web
# Task/uot 113267

## Description
Adds the Jira feedback module to allow users to submit feedback that will be posted as an issue on the Advana Helpdesk 

## Ticket
task/UOT-113267

## Testing Instructions
* Update your backend .env with the following vars and contact me for values:
JIRA_USERNAME=
JIRA_PASSWORD=
JIRA_DOMAIN=
JIRA_PROJECT_KEY=
JIRA_RATING_ID=
JIRA_FEEDBACK_TYPE=
* Build the application after setting env vars
* Click on user feedback in the side navigation bar and submit feedback (you need to be on AnyConnect when feedback is submitted)
* Check the given Jira domain to ensure feedback was posted
* Go to the admin panel and toggle Jira user feedback 
* Click on user feedback in the side navigation bar and ensure old feedback modal pops up
* Admin toggle again and make sure the new feedback modal pops up
